### PR TITLE
Cardano db sync watcher

### LINKED ISSUE #65: Implement watcher based on PR #13.
Ref: #13 

Useful query to check the sync state:
https://github.com/IntersectMBO/cardano-db-sync/blob/master/doc/interesting-queries.md#sync-progress-of-db-sync

### PR DESCRIPTION
**Pull request by albertodvp**
Closes #65

### CODE REVIEW

File: cardanow-ts/cardano-db-sync-watcher.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    } else {
+      console.log(`Epoch number does not match the expected value. epochNo: ${epochNo}, expected: ${epoch}`);
+      return false;
+    }
+
+  } catch (e) {
+    console.error(e);
+
+    // Always return true even if there's an error
+    return false;
```
Comment (aciceri) [inline cardanow-ts/cardano-db-sync-watcher.ts:74]: True or false? In case why always true?

File: config/docker-compose.yaml [hunk 2]
```diff
@@ -44,9 +55,73 @@ services:
       - "--match"
       - "*"
 
+  # TODO move variables out
```
Comment (aciceri) [inline config/docker-compose.yaml:58]: Where? What do you mean?

File: flake.nix [hunk 3]
```diff
@@ -3,17 +3,17 @@
   inputs = {
     cardano-node.url = "github:intersectmbo/cardano-node";
     cardano-configurations = {
-      url = "github:input-output-hk/cardano-configurations?rev=fa19a63c9a2050940583b10e9f39ed02e3cf306f";
+      url = "github:input-output-hk/cardano-configurations?rev=6d7fac44d6a877ae9a3cf5eeda163ae539b17774";
```
Comment (aciceri) [inline flake.nix:6]:
```suggestion
      url = "github:input-output-hk/cardano-configurations/6d7fac44d6a877ae9a3cf5eeda163ae539b17774";
```

You can rewrite it like this if you prefer, nothing should change I believe.

File: nix/packages/cardanow.nix [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
     cardanow-ts
     cleanup-local-data
     curl
     git
-    cardano-cli
-    mithril-client-cli
     jq
+    mithril-client-cli
     openssh
+    postgresql_14
```
Comment (aciceri) [inline nix/packages/cardanow.nix:15]: This has to corresponds with the postgres service running in docker, right? Do we want to add a comment perhaps?

File: nix/shell.nix [hunk 5]
```diff
@@ -32,14 +32,20 @@
...
         inputs'.cardano-node.packages.cardano-cli
         inputs'.mithril.packages.mithril-client-cli
         jq
         nil
         nixos-rebuild
         nodejs_20
+        nodePackages.bash-language-server
+        nodePackages.vscode-langservers-extracted
+        nodePackages.prettier
+        postgresql_14
```
Comment (aciceri) [inline nix/shell.nix:46]: As above

File: config/docker-compose.yaml (hunk 2 above)
Comment (albertodvp) [reply config/docker-compose.yaml:58]: I've already done that, I forgot to remove the comment. Thanks!

**Pull request merged**